### DISCUSSION TITLE
Issues with auto ortho

### DISCUSSION THREAD

**Discussion by CptAirwolf** [kubilus1/autoortho#218] [General]
I have looked and not found anything discussed about this hence why the topic. I have a problem where all my files are extracted into the custom scenery folder yet the auto ortho program just simply wont detect them. I downloaded Europe and had to manually extract the files myself. I have every file in the folder and the scenery packs is setup. But when loading up the auto ortho program it says europe was not set up. And then while trying to run the program it exits. Any help?

**Comment by Andreweag** [comment]
On the main autoortho page make sure that you have linked your custom scenery folder.

**Comment by CptAirwolf** [comment]
Yes I do. After re installing everything im having the Zip File issue.
